purchase_order_type/tests/test_purchase_order_type.py

```diff
@@ -65,3 +65,14 @@ def _create_purchase(self, line_products):
             }
         )
         return purchase
+
+    def test_purchase_order_change_company(self):
+        order = self.po_obj.new({"partner_id": self.partner1.id})
+        self.assertEqual(order.order_type, self.type1)
+        order._onchange_company()
+        self.assertFalse(order.order_type)
+
+    def test_purchase_order_type_company_error(self):
+        order = self.po_obj.create({"partner_id": self.partner1.id})
+        self.assertEqual(order.order_type, self.type1)
+        self.assertEqual(order.company_id, self.type1.company_id)
```

Comment:
You should test for error like this,
https://github.com/ecosoft-odoo/sale-workflow/blob/13.0-enh-sale_order_type/sale_order_type/tests/test_sale_order_type.py#L130